Lombiq.NodeJs.Extensions.Samples/package.json

```diff
@@ -1,20 +1,8 @@
 {
   "private": true,
-  "devDependencies": {
-    "nodejs-extensions": "link:..\\Lombiq.NodeJs.Extensions",
-    "npm-run-all": "^4.1.5"
-  },
   "scripts": {
-    "dotnet-prebuild": "pnpm build",
-    "dotnet-postclean": "pnpm clean",
-    "build": "npm-run-all --parallel build:styles build:styles:args",
-    "build:styles": "npm explore nodejs-extensions -- pnpm build:styles",
-    "build:styles:args": "npm explore nodejs-extensions -- pnpm build:styles:args --source=NonDefaultScssFolder --target=wwwwNonDefaultCssFolder",
-    "clean": "npm-run-all --parallel clean:styles clean:styles:args",
-    "clean:styles": "npm explore nodejs-extensions -- pnpm clean:styles",
-    "clean:styles:args": "npm explore nodejs-extensions -- pnpm clean:styles:args --target=wwwwNonDefaultCssFolder",
-    "watch": "npm-run-all --parallel watch:styles watch:styles:args",
-    "watch:styles": "npm explore nodejs-extensions -- pnpm watch:styles",
-    "watch:styles:args": "npm explore nodejs-extensions -- pnpm watch:styles:args --source=NonDefaultScssFolder --target=wwwwNonDefaultCssFolder"
```

Comment:
Why not keep these here for demonstration?

Reply:
Well, we have all the scripts this project needs below. The scripts without `:args` are referenced in the `Samples.NuGet` project.

Reply:
But why not also have `build:styles` and the likes?

Reply:
Because those use the default locations for SCSS and CSS files. This sample project only uses non-default paths. As discussed in JIRA, we don't need support for both at once.

Reply:
I'm arguing that having both `build:styles` AND `build:styles:args` scripts is NOT a common case, because it would imply that the consumer project has both the default AND a non-default location with SCSS files.

As discussed, we actually don't provide MSBuild integration for BOTH at the same time. You either use the default locations _Assets/Styles_ and _wwwroot/css_ or you use something else. In the former case, you'd have:

    "build:styles": "npm explore nodejs-extensions -- pnpm build:styles"

and in the latter:

    "build:styles:args": "npm explore nodejs-extensions -- pnpm build:styles:args --source=NonDefaultScssFolder --target=wwwwNonDefaultCssFolder"

It's either or.

Reply:
I see. How about having just `build:styles`, `clean:styles` and `watch:styles` though?

Reply:
That sounds sensible. I'll do that.